latestnumbers/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for latestnumbers project.
+
+Generated by 'django-admin startproject' using Django 4.1.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-^(!x+#o@qh+9^8y$!$5fbwr3gmv1gw==zi!g^8icn^xruitboi"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```

Comment:
I don't see it here, but we would want a settings file (or modify this one to fetch from env) that allows us to set the secret key and allowed hosts, and also defaults to debug mode off.